models/utils.js

```diff
@@ -10,33 +10,52 @@ exports.checkParametricFormat = (value, column, table) => {
     WHERE
         %2$s=%1$s;
     `, value, column, table)
+
     return db
         .query(queryString)
-        .then(() => {
-            return Promise.resolve();
+        .then(({ rows }) => {
+            if (rows.length !== 0) {
+                return Promise.resolve();
+            } else {
+                return Promise.reject({ status: 404, message: '404: parametric endpoint not found' })
```

Comment:
You could make this shorter by getting rid of the Promise.resolve() (because it's just an either/or situation). So you could say if the `rows.length` is 0, then reject it.